code/modules/mob/living/carbon/human/species/station/seromi.dm

```diff
@@ -56,7 +56,7 @@
 	mob_size = MOB_SMALL
 	pass_flags = PASSTABLE
 	holder_type = /obj/item/weapon/holder/human
-	short_sighted = 1
+	short_sighted = 0
```

Comment:
a

Reply:
did it fuck up. Sorry.

Reply:
https://cdn.discordapp.com/attachments/729321033475293204/733244394538270750/unknown.png

no it didn't? I reverted those changes

Reply:
git is flawless

Reply:
I have to agree on that one